### DISCUSSION TITLE
Script Netbox Location

### DISCUSSION THREAD

**Discussion by TomasMondim** [netbox-community/netbox#11083] [General]
Hello, im trying to add a location to my device, but i dont know how to do it, in this way is getting errored, any help?
<img width="431" alt="Capture" src="https://user-images.githubusercontent.com/73384112/205237144-7753ee38-bc1b-410c-84bd-bba68ba74d73.PNG">

**Comment by eronlloyd** [comment]
By location do you mean an existing Location record in Netbox? For that you would use an [ObjectVar](https://docs.netbox.dev/en/stable/customization/custom-scripts/#objectvar) in your script.
